src/extras/amd.js

```diff
@@ -69,6 +69,13 @@
     systemRegister.apply(this, arguments);
   };
 
+  const prepareImport = systemPrototype.prepareImport;
+  systemPrototype.prepareImport = function() {
+    // Reset "currently executing script"
+    amdDefineDeps = null;
```

Comment:
Seems good. Can we move this into the beginning of `instantiate` rather though? So to get as close to the time of script injection as possible.

Reply:
👍 updated